modules/python3qt/pythoneditorwidget.cpp

```diff
@@ -198,9 +198,16 @@ PythonEditorWidget::PythonEditorWidget(QWidget* parent, InviwoApplication* app)
         settings.beginGroup(objectName());
         QString lastFile = settings.value("lastScript", "").toString();
         if (!lastFile.isEmpty()) loadFile(lastFile.toLocal8Bit().constData(), false);
-        
+        else {
+            QString src = settings.value("source", "").toString();
+            if(src.length()!=0){
+                pythonCode_->setPlainText(src);
+                script_.setSource(utilqt::fromQString(src)); // or use localQString? 
```

Comment:
what is the reason for the comment?

Reply:
Comment is because I forgot to remove it. Not sure which to use.

Reply:
preferably `utilqt::fromQString()` and `utilqt::toQString()`